src/Kernel32/Kernel32+PRODUCT_SUITE.cs

```diff
@@ -90,7 +90,22 @@ public enum PRODUCT_SUITE : short
             /// <summary>
             /// Windows Home Server is installed.
             /// </summary>
-            VER_SUITE_WH_SERVER = unchecked((short)0x00008000)
+            VER_SUITE_WH_SERVER = unchecked((short)0x00008000),
+
+            /// <summary>
+            /// AppServer mode is enabled.
+            /// </summary>
+            VER_SUITE_MULTIUSERTS = unchecked((short)0x00020000),
+
+            /// <summary>
+            /// Windows NT server is installed.
+            /// </summary>
+            VER_SERVER_NT = unchecked((short)0x80000000),
+
+            /// <summary>
+            /// Windows NT Workstation is installed.
+            /// </summary>
+            VER_WORKSTATION_NT = unchecked((short)0x40000000),
```

Comment:
These enum values are all `0` because you're truncating the numbers you're providing to just 16 bits, and the last 16 bits are all zero. Clearly these could never be used in this enum which apparently is intentionally sized at 16 bits to fit the `WORD` size used in the methods that take this enum. 
I've removed these new values.